DM-1357 Add LWM2M 1.1 information to What's New (#56)
SoftwareAG/c8y-releasenotes: content/release-10-15-0/whatsnew-10-15-0.md

For information on installation and operations, see the *Digital Twin Manager -  Installation & operations guide*.

### Device Management

> ### Revised LWM2M device registration

Reviewer: This and the following headings should have heading level 4 ####

Author: Done

> The LWM2M agent now provides now registration wizards for both single and bulk LWM2M device registration. These dialogs allows the user to conveniently perform single and bulk registrations of LWM2M devices. They also perform input validation and detect common errors before the device is created. See also [LWM2M > Registering LWM2M devices](https://cumulocity.com/guides/protocol-integration/lwm2m/##register-device) in the *Protocol integration guide*.

Reviewer: ```suggestion
The LWM2M agent now provides registration wizards for both single and bulk LWM2M device registration. These dialogs allows the user to conveniently perform single and bulk registrations of LWM2M devices. They also perform input validation and detect common errors before the device is created. See also [LWM2M > Registering LWM2M devices](https://cumulocity.com/guides/10.15.0/protocol-integration/lwm2m/#register-device) in the *Protocol integration guide*.
```

For information on installation and operations, see the *Digital Twin Manager -  Installation & operations guide*.

### LWM2M 1.1 support

We are now supporting the following LWM2M 1.1 features:

- LWM2M Send
- Composite Operations (read, write, observe, cancel observation)
- New LWM2M serialization formats: CBOR, SenML JSON, SenML CBOR
- New LWM2M data formats: Core Link, Unsigned Integers

> For more information on our 1.1 support, see our [LwM2M protocol integration guide](http://localhost:58835/guides/protocol-integration/lwm2m/).

Reviewer: The links point to your local environment.

Author: Whoopsie!

<video width="99%" controls>
  <source src="/images/release-notes/lwm2m-bulk-reg.mp4" type="video/mp4" />
</video>

> #### LWM2M 1.1 support

Reviewer 2: `####` -> `###`

Author: Fixed

> We are now supporting the following LWM2M 1.1 features:

Reviewer: ```suggestion
We now support the following LWM2M 1.1 features:
```

- LWM2M Send
- Composite Operations (read, write, observe, cancel observation)
- New LWM2M serialization formats: CBOR, SenML JSON, SenML CBOR
- New LWM2M data formats: Core Link, Unsigned Integers

> For more information on our 1.1 support, see our [LwM2M protocol integration guide](https://cumulocity.com/guides/protocol-integration/lwm2m/).

Reviewer: ```suggestion
For more information on our 1.1 support, refer to [LWM2M](https://cumulocity.com/guides/10.15.0/protocol-integration/lwm2m/) in the *Protocol integration guide*.
```

Reviewer: Perhaps add the section here?

<video width="99%" controls>
  <source src="/images/release-notes/lwm2m-bulk-reg.mp4" type="video/mp4" />
</video>

#### LWM2M 1.1 support

We are now supporting the following LWM2M 1.1 features:

- LWM2M Send
- Composite Operations (read, write, observe, cancel observation)
- New LWM2M serialization formats: CBOR, SenML JSON, SenML CBOR
- New LWM2M data formats: Core Link, Unsigned Integers

For more information on our 1.1 support, see our [LwM2M protocol integration guide](/guides/protocol-integration/lwm2m/).

> #### LWM2M time timestamp conversion

Reviewer 2: LWM2M device object timestamp conversion?

Author: fixed

> If a timestamp (resources 5518 and 6050, resource 5 for object 6) is received together with resources of the same object, this timestamp can be used as creation timestamp for measurements, events and alarms. See [Advanced settings](/guides/protocol-integration/lwm2m/#advanced-settings) for details.

Reviewer 2: Maybe you can add it needs to be enabled.
See [Advanced settings](...) for details and to enable during device registration.
